dnfdaemon-client/wrappers/dbus_transaction_package_wrapper.hpp

```diff
@@ -43,6 +43,8 @@ class DbusTransactionPackageWrapper {
     DbusPackageWrapper get_package() const { return package; }
     libdnf::transaction::TransactionItemAction get_action() const { return action; }
     libdnf::transaction::TransactionItemReason get_reason() const { return reason; }
+    // TODO(jmracek) Fully integrate get_replaces()
```

Comment:
I don't mind this as a note for yourself if you're going to do this in near future, but otherwise a better description would be nice, because it's not clear at all what you mean by this.

Reply:
Fixed